pkg/controller/operators/catalog/operator.go

```diff
@@ -1168,16 +1168,11 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 					return errorwrap.Wrapf(err, "error parsing step manifest: %s", step.Resource.Name)
 				}
 
-				// Update UIDs on all CSV OwnerReferences
-				updated, err := o.getUpdatedOwnerReferences(cr.OwnerReferences, plan.Namespace)
-				if err != nil {
-					return errorwrap.Wrapf(err, "error generating ownerrefs for clusterrole %s", cr.GetName())
-				}
-				cr.OwnerReferences = updated
-
 				// Attempt to create the ClusterRole.
 				_, err = o.OpClient.KubernetesInterface().RbacV1().ClusterRoles().Create(&cr)
 				if k8serrors.IsAlreadyExists(err) {
+					// if we're updating, point owner to the newest csv
+					cr.Annotations[ownerutil.OwnerKey] = step.Resolving
```

Comment:
Isn't this supposed to be a label?

Reply:
It should be a label, yes. 

We're not currently GCing at all, so switching it should be fine as well. We do need to have an api similar to ownerrefs though, and GC these ourselves (or implement GC in kube)